src/navigation/NavBar.js

```diff
@@ -26,77 +25,66 @@ const NavLink = ({
   const href = `#${to}`
   const active = exact ? hash === href : hash && hash.startsWith(href)
   const anchorClass = classnames({
-    'bg-white-10': active,
+    'bg-white-10 navbar-item-active': active,
     'o-50 no-pointer-events': disabled
-  }, ['dib db-l pv3 white no-underline f5 hover-bg-white-10 tc'])
+  }, ['dib db-l pt2 pb3 pv3-l white no-underline f5 hover-bg-white-10 tc bb bw2 bw0-l b--navy'])
   const svgClass = classnames({
     'o-100': active,
     'o-50': !active
   }, ['fill-current-color'])
 
   return (
     // eslint-disable-next-line jsx-a11y/anchor-is-valid
-    <a href={disabled ? null : href} className={anchorClass} style={{ borderLeft: active ? '5px solid rgba(201, 210, 215, .4)' : '' }} role='menuitem' title={children}>
-      <span className={`dib ${open ? 'dt-l' : ''}`}>
-        <span className={`dib dtc-l v-mid ${open ? 'pl3 pl5-l' : 'ph3'}`} style={{ width: 50 }}>
+    <a href={disabled ? null : href} className={anchorClass} role='menuitem' title={children}>
+      <div className='db ph2 pv1'>
+        <div className='db'>
           <Svg width='50' className={svgClass} />
```

Comment:
Forgot about this, we should do `role="img" aria-labelledby="IPFS logo"

Reply:
Better yet, `role="presentation"`

Reply:
👍  Added `role='presentation'` to divs surrounding icons: see lines 40 and 60. Is this what you're thinking?

Reply:
@rafaelramalho19 - per our chat moved these directly into the img tags. Look good to you?

Reply:
It's perfect, thank you 😄